### PR TITLE
Added end-to-end test with analyzers disabled #3177

### PR DESCRIPTION
**Pull request by joachimmetz**
Added end-to-end test with analyzers disabled #3177

### CODE REVIEW

File: config/end-to-end.ini [hunk 1]
```diff
@@ -37,6 +37,14 @@ output_format=dynamic
 reference_output_file=test_data/end_to_end/extract_and_output.log
 source=test_data/vsstest.qcow2
 
+[extract_and_output_no_hahsers1]
```
Comment (rgayon) [inline config/end-to-end.ini:40]: s/hs/sh/

File: config/end-to-end.ini [hunk 2]
```diff
@@ -37,6 +37,14 @@ output_format=dynamic
...
+case=extract_and_output
+extract_options=--no-vss --hashers=none
+output_file=extract_and_output_no_hahsers1.log
```
Comment (rgayon) [inline config/end-to-end.ini:43]: same

File: config/end-to-end.ini (hunk 1 above)
Comment (joachimmetz) [reply config/end-to-end.ini:40]: ack, good catch

File: config/end-to-end.ini (hunk 2 above)
Comment (joachimmetz) [reply config/end-to-end.ini]: done

**Comment by joachimmetz** [comment]
@rgayon feedback addressed, PTAL

**Pull request merged**